tests/dapr_tests.mk

```diff
@@ -89,7 +93,15 @@ $(foreach ITEM,$(E2E_TEST_APPS),$(eval $(call genTestAppImageBuild,$(ITEM))))
 define genTestAppImagePush
 .PHONY: push-e2e-app-$(1)
 push-e2e-app-$(1): check-e2e-env
+ifeq ($(TARGET_OS),windows)
+ifeq (,$(wildcard $(E2E_TESTAPP_DIR)/$(1)/$(DOCKERFILE)))
```

Comment:
If the push is dependent on the existence of Dockerfile-windows, and you still have one for actorjava, won't this still be enabled? Or am I missing something?

Reply:
Oh, I have the logic backwards in my head. You push only when an app specific Dockerfile-windows **doesn't** exist. I get it now. Maybe a comment to explain the logic a little bit?